[2024-04-15T09:57:42.346+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-15T09:57:42.615+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prioritas1_no2.check_even_odd scheduled__2024-04-13T00:00:00+00:00 [queued]>
[2024-04-15T09:57:42.662+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prioritas1_no2.check_even_odd scheduled__2024-04-13T00:00:00+00:00 [queued]>
[2024-04-15T09:57:42.664+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 6
[2024-04-15T09:57:42.749+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): check_even_odd> on 2024-04-13 00:00:00+00:00
[2024-04-15T09:57:43.108+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=62) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-15T09:57:43.119+0000] {standard_task_runner.py:63} INFO - Started process 76 to run task
[2024-04-15T09:57:43.107+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'prioritas1_no2', 'check_even_odd', 'scheduled__2024-04-13T00:00:00+00:00', '--job-id', '269', '--raw', '--subdir', 'DAGS_FOLDER/nomer2_dag.py', '--cfg-path', '/tmp/tmpjbnyexdp']
[2024-04-15T09:57:43.136+0000] {standard_task_runner.py:91} INFO - Job 269: Subtask check_even_odd
[2024-04-15T09:57:43.323+0000] {task_command.py:426} INFO - Running <TaskInstance: prioritas1_no2.check_even_odd scheduled__2024-04-13T00:00:00+00:00 [running]> on host 2354a2c4893f
[2024-04-15T09:57:44.246+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abdan' AIRFLOW_CTX_DAG_ID='prioritas1_no2' AIRFLOW_CTX_TASK_ID='check_even_odd' AIRFLOW_CTX_EXECUTION_DATE='2024-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-13T00:00:00+00:00'
[2024-04-15T09:57:44.251+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-15T09:57:44.319+0000] {logging_mixin.py:188} INFO - Gagal mengambil jumlah dari XCom. Melewati task.
[2024-04-15T09:57:44.322+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-15T09:57:44.323+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-15T09:57:44.344+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=prioritas1_no2, task_id=check_even_odd, execution_date=20240413T000000, start_date=20240415T095742, end_date=20240415T095744
[2024-04-15T09:57:45.841+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-15T09:57:45.890+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-15T09:57:45.989+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
